### DISCUSSION TITLE
可能開發免安裝版嗎？

### DISCUSSION THREAD

**Discussion by dummyvoid** [rime/home#1656] [Ideas]
有些电脑无法自由安装软件，可能开发免安装版吗？

**Comment by cvlot** [comment]
其实Rime, 本身就可以绿化便携版本

**Comment by fxliang** [comment]
https://github.com/fxliang/rime.toy/actions/runs/11757288717

这样可能才算绿色（玩具）

**Reply by cvlot** [reply]
这个玩具版, 就只是没有ui界面吗, 我感觉少了好多功能, 每次更改配置yaml, 不需要重新部署吗?

**Reply by fxliang** [reply]
> 这个玩具版, 就只是没有ui界面吗, 我感觉少了好多功能, 每次更改配置yaml, 不需要重新部署吗?

用GUI分支的产物，带候选界面的呀，我用来做ui重构的（当然只是雏形），我还做了托盘右键部署的
